std/assembly/math.ts

```diff
@@ -992,6 +992,13 @@ export namespace NativeMath {
     }
   }
 
+  @inline
+  export function signbit(x: f64): bool {
+    // In ECMAScript all NaN values are indistinguishable from each other
+    // so we need handle NaN and negative NaN in similar way
+    return <bool>(<i32>(reinterpret<u64>(x) >>> 63 != 0) & <i32>(x == x));
```

Comment:
Similar to the f32 version, the `!= 0` instruction can be saved here as well.

Reply:
I'm not sure. Because without `!= 0` `u64` will be cast to i32 anyway

Reply:
Oh, actually `!= 0` doesn't matter because it has to convert from 64 to 32 bits anyway.